src/libical-glib/api/i-cal-time.xml

```diff
@@ -143,20 +145,20 @@
         <parameter type="const ICalTime *" name="a" annotation="in, transfer none" comment="The #ICalTime to be compared"/>
         <parameter type="const ICalTime *" name="b" annotation="in, transfer none" comment="The #ICalTime to be compared"/>
         <returns type="gint" comment="-1, 0, or 1 to indicate that a less than b, a==b or a larger than b." />
-        <comment xml:space="preserve">Return -1, 0, or 1 to indicate that a less than b, a==b or a larger than b.</comment>
+        <comment xml:space="preserve">Return -1, 0, or 1 to indicate that a is less than b, a equals b, or a is greater than b.</comment>
     </method>
     <method name="i_cal_time_compare_date_only" corresponds="icaltime_compare_date_only" since="1.0">
         <parameter type="const ICalTime *" name="a" annotation="in, transfer none" comment="The #ICalTime to be compared"/>
         <parameter type="const ICalTime *" name="b" annotation="in, transfer none" comment="The #ICalTime to be compared"/>
         <returns type="gint" comment="-1, 0, or 1 to indicate that a less than b, a==b or a larger than b." />
-        <comment xml:space="preserve">like i_cal_time_compare, but only use the date parts.</comment>
+        <comment xml:space="preserve">Like i_cal_time_compare, but only use the date parts.</comment>
     </method>
     <method name="i_cal_time_compare_date_only_tz" corresponds="icaltime_compare_date_only_tz" since="1.0">
         <parameter type="const ICalTime *" name="a" annotation="in, transfer none" comment="The #ICalTime to be compared"/>
         <parameter type="const ICalTime *" name="b" annotation="in, transfer none" comment="The #ICalTime to be compared"/>
 	<parameter type="ICalTimezone *" name="zone" annotation="nullable" comment="The target timezone"/>
         <returns type="gint" comment="-1, 0, or 1 to indicate that a less than b, a==b or a larger than b." />
-        <comment xml:space="preserve">like i_cal_time_compare_tz, but only use the date parts.</comment>
+        <comment xml:space="preserve">Like icaltime_compare, but only use the date parts; accepts timezone.</comment>
```

Comment:
```suggestion
        <comment xml:space="preserve">Like i_cal_time_compare_tz(), but only use the date parts; accepts timezone.</comment>
```
Please keep it with the libical-glib namespace :slightly_smiling_face:

Reply:
Good catch, thanks! I don't know of a function called `i_cal_time_compare_tz` though, do you mean `i_cal_time_compare`? I know it was in the earlier version, but afaik it doesn't exist, maybe it's a deprecation? Also, I don't see the use of parens anywhere else, so to be consistent with the rest I think that it should not include that. Does this look better?

```suggestion
        <comment xml:space="preserve">Like i_cal_time_compare, but only use the date parts; accepts timezone.</comment>
```

Reply:
Indeed `i_cal_time_compare_tz` doesn't exist so it is probably indeed `i_cal_time_compare`. All the reference to functions in libical-glib should be using foo_bar() as this documentation is built using GTK-Doc and thus should be following the [supported syntax](https://developer.gnome.org/gtk-doc-manual/stable/documenting_syntax.html). In this case, having a foo_bar() will allow GTK-Doc to automatically generate a link to the right function.

Reply:
Should I change the other function references to include the `()` token then too? Thanks for helping me with this, I'm new to GTK-Doc.

(A suggestion so I can stay within the GitHub system)
```suggestion
        <comment xml:space="preserve">Like i_cal_time_compare(), but only use the date parts; accepts timezone.</comment>
```